src/main/java/service/FileReaderService.java

```diff
@@ -0,0 +1,7 @@
+package service;
+
+import java.util.List;
+
+public interface FileReaderService {

```

Comment:
better have one `interface FileService` with `readFile `and `writeToFile` methods